frontend/tests/unit/fragments/public-schools/school-totals-test.js

```diff
@@ -0,0 +1,271 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { run } from '@ember/runloop';
+
+import SchoolTotals from 'labs-ceqr/fragments/public-schools/SchoolTotals';
+import Building from 'labs-ceqr/fragments/public-schools/Building';
+
+module('Unit | Fragment | SchoolTotals', function(hooks) {
+  setupTest(hooks);
+
+  test('#buildings returns correct filters list', function(assert) {
+    let buildings = run(() => [
+      Building.create({
+        level: 'hs',
+        district: 1,
+        subdistrict: 2
+      }),
+      Building.create({
+        level: 'ps',
+        district: 1,
+        subdistrict: 2
+      }),
+      Building.create({
+        level: 'ps',
+        district: 1,
+        subdistrict: 3
+      }),
+      Building.create({
+        level: 'is',
+        district: 1,
+        subdistrict: 2
+      }),
+    ]);
+
+    let hs_school_totals = run(() => SchoolTotals.create({
+      allBuildings: buildings,
+      level: 'hs'
+    }));
+
+    assert.deepEqual(hs_school_totals.buildings, [
+      Building.create({
+        level: 'hs',
+        district: 1,
+        subdistrict: 2
+      })
+    ]);
```

Comment:
So is the behavior we're testing here that it `.buildings` returns a certain filtered subset? Only `hs`?

Reply:
Yes that's what we're testing. I can add comments above the test to explain that